game/scripts/npc/abilities/talents/earthshaker_talents.txt

```diff
@@ -0,0 +1,71 @@
+"DOTAAbilities"
+{
+//=================================================================================================================
+	// Enchant Totem Cooldown
+	//=================================================================================================================
+	"special_bonus_unique_earthshaker"
+	{
+		// General
+		//-------------------------------------------------------------------------------------------------------------
+		"ID"					"6035"														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
+		"AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
+		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+		
+		// Special
+		//-------------------------------------------------------------------------------------------------------------
+		"AbilitySpecial"
+		{
+			"01"
+			{
+				"var_type"					"FIELD_INTEGER"
+				"value"				"2"
+			}
+		}
+	}
+
+	//=================================================================================================================
+	// Echo Slam Damage
+	//=================================================================================================================
+	"special_bonus_unique_earthshaker_2"
+	{
+		// General
+		//-------------------------------------------------------------------------------------------------------------
+		"ID"					"6425"														// unique ID number for this ability.  Do not change this once established or it will invalidate collected stats.
+		"AbilityType"					"DOTA_ABILITY_TYPE_ATTRIBUTES"
+		"AbilityBehavior"				"DOTA_ABILITY_BEHAVIOR_PASSIVE"
+		
+		// Special
+		//-------------------------------------------------------------------------------------------------------------
+		"AbilitySpecial"
+		{
+			"01"
+			{
+				"var_type"					"FIELD_INTEGER"
+				"value"				"100" //OAA
```

Comment:
I wouldn't buff the talent because we now know why Echo Slam felt underwhelming at 15-20 min (initial damage isn't increasing because of the wrong KV). And fixing the KV is already a buff.